i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Callable
+
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class ConformerPositionwiseFeedForwardV1Config(ModelConfiguration):
+    input_dim: int
+    """input dimension"""
+    hidden_dim: int
+    """hidden dimension (normally set to 4*input_dim as suggested by the paper)"""
+    dropout: float
```

Comment:
Do we want to have a default value for dropout?

Reply:
I see that @JackTemaki commented in another PR for not having any defaults, for dropout I think it's OK. for the activation function I think there should be a default, as it's part of the paper and usually not something people "tune" as a hyper-parameter.

Reply:
Okay, I have just added nn.functional.silu as the default activation